sunbird_mcp/src/config.py

```
@@ -0,0 +1,136 @@
+"""Configuration settings for the Sunbird MCP Server."""
+from typing import Dict, List
+from pydantic import Field, field_validator
+from pydantic_settings import BaseSettings
+
+class Settings(BaseSettings):
+    """Application settings and configuration."""
+    
+    # API Configuration
+    API_BASE_URL: str = Field(
+        default="https://diksha.gov.in",#This is the base url for the sunbird api
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)